challenges.yml

```
@@ -193,3 +193,189 @@ stimulus_values:
         d: |
           `this.values.colorSelected`;
       correct: c
+value_change_callback:
+  question: |
+    Supose we have a Stimulus controller like this one:
+
+    ```js
+    export default class extends Controller {
+        static targets = ['selectItem']
```

```suggestion
        static targets = ['selectItem'];
```

Do we need semicolon here?